rocket_twin/systems/control/controller.mo

```diff
@@ -1,8 +1,18 @@
 model controller
   input Real ti;
+  input Real weight;
+  parameter Real weight_max;
+  parameter Real t0;
   output Real w;
+  output Boolean engine_on;
 equation
-  if (ti < 5.) then
+  if (ti < t0) then
+    engine_on = false;
+  else
+    engine_on = true;
+  end if;
+
+  if (weight < weight_max and engine_on == false) then
```

Comment:
there is something to do to manage valve status and connection status instead of engine status : an idea ?